.github/workflows/continuous-integration.yml

```diff
@@ -87,17 +81,21 @@ jobs:
     #
     # Build libraries, examples and tests
     #
-    - name: Build libraries
+    - name: Build libraries and tests
       run: |
         $FC --version
-        make -j4 libs
-    #
-    # Run examples and tests
-    #
-    - name: Build and run examples and tests
-      run: make -j4 tests
-    #
-    # Compare the results
-    #
-    - name: Compare the results
-      run: make -j4 check
+        cmake -S . -B build \
+            -DCMAKE_Fortran_COMPILER=$FC \
+            -DCMAKE_Fortran_FLAGS="$FCFLAGS" \
+            -DRRTMGP_DATA_VERSION=$RRTMGP_DATA_VERSION \
+            -DFP_MODEL=$FP_MODEL \
```

Comment:
@makepath-alex Can I ask why you moved $FP_MODEL to the CMake configuration while leaving $RTE_USE_CBOOL as a compiler flag? The two choices are implemented in the same way (with \#IFDEFS in the code).

Reply:
Missed one :D